### DISCUSSION TITLE
请问useTable怎么实现后台分页

### DISCUSSION THREAD

**Discussion by hpp1106geek** [vbenjs/vue-vben-admin#789] [Q&A]
设置了
pagination:{
   total:100,
   pageSize:10,
   curent:1,
},           
没有效果

**Comment by IFnGSiYu** [comment]
看下后台请求的分页传参字段名是不是一致的，不是的就在settings/componentSetting改下就行

**Comment by atorber** [comment]
同问，实测在示例表格中当从第一页切换到其他页时页面为空，因为示例接口每次返回的是单次请求的数据，永远都只有一页数据，前端采用的是覆盖式更新数据，那么怎么实现前端缓存更新呢

![image](https://user-images.githubusercontent.com/19552906/200789356-1e4df968-c510-4bfd-9d9d-b05302282260.png)
